src/app/dashboard/user-card/user-card.component.html

```diff
@@ -0,0 +1,80 @@
+<div class="ui cards">
+  <div class="card">
+    <div class="content">
+      <div class="header">{{ user.firstName + " " }}{{ user.lastName }}</div>
+    </div>
+    <div class="extra content">
+      <button
+        (click)="showViewModal = true"
```

Comment:
Is this a boolean check for truth? The syntax is wrong. Please fix this here and in all the other places too.